# bspline optimizer (optimization/)
lambda_smooth: 1.0
lambda_collision: 10.5
lambda_feasibility: 0.1
lambda_fitness: 1.0
dist0: 0.5
swarm_clearance: 0.1
max_vel: 2.0
max_acc_: 3.0
order_: 3